daphne/src/testing.rs

```diff
@@ -773,12 +774,12 @@ impl DapLeader<BearerToken> for MockAggregator {
         let mut leader_state_store_mutex_guard = self
             .leader_state_store
             .lock()
-            .map_err(|e| DapError::Fatal(e.to_string()))?;
+            .map_err(|e| fatal_error!(err = e))?;
         let leader_state_store = leader_state_store_mutex_guard.deref_mut();
 
         let leader_state = leader_state_store
             .get(task_id)
-            .ok_or_else(|| DapError::fatal("collect job not found for task_id"))?;
+            .ok_or_else(|| fatal_error!(err = "collect job not found for task_id", %task_id))?;
```

Comment:
I am not familiar with the `%` syntax here. Please document it along with the macro definition.